PowerEditor/installer/nativeLang/german.xml

```diff
@@ -1115,7 +1115,7 @@ Translation note:
 					<Item id="6310" name="Einrückung verwendet:"/>
 					<Item id="6311" name="Tab Zeichen"/>
 					<Item id="6510" name="Standard verwenden"/>
-					<Item id="6512" name="Rückschritttaste entfernt Einrückungen, anstatt einzelne Leerzeichen zu entfernen"/>
+					<Item id="6512" name="Die Rückschrittaste U+232B entfernt Einrückungen statt einzelne Leerzeichen"/>
```

Comment:
I would just call it Rücktaste which seems the most common for me, U+232B is imo not needed, so:

"Rücktaste entfernt Einrückungen anstatt einzelner Leerzeichen"

Reply:
Are there more German speakers who can provide the opinion regarding this change?

Reply:
LEO.org validates it, too. I think Rücktaste is the correct and most common word for the backspace key.

https://dict.leo.org/englisch-deutsch/backspace

![grafik](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/82021174/d16e7812-c4e2-4b73-8b0a-5b328782e695)

Reply:
As a German I say Backspace-Taste (which is the name in the according Wikipedia article too). With Rücktaste I would know what is meant. Rückschritttaste is rather uncommon.